admin-dev/themes/default/js/bundle/product/form.js

```diff
@@ -44,12 +44,6 @@ $(document).ready(function() {
   tags.init();
   rightSidebar.init();
   recommendedModules.init();
-  BOEvent.emitEvent("Product Categories Management started", "CustomEvent");
```

Comment:
@Quetzacoalt91 I could keep theses events, but I'm not sure this works as we expect: I expect an event to be asynchronous and to work even if I minify/compile my assets => it's not the case right now :/

Reply:
IMO, I would at least keep the event emission, even if it is (currently?) synchronous.